README.md

```diff
@@ -45,6 +45,23 @@ The following `npm` scripts are defined for convenience:
 
 The TetraScience Agents produce a number of structured logs and metrics which provide insight into the execution of the agents. There is not, however, a productized solution for monitoring these metrics.  This repository is designed to be a starting point to leverage AWS CloudWatch to build a monitoring system (metrics and alarms) for your agents.
 
+### Metrics
+
+| Metric | Pattern | Dimensions | Value |
+| ------ | ------- | ---------- | ----- |
+| AgentMonitoring/PerPathScanDuration | { $.event.type = "agents.filelog.scanCompleted.v1" } | - orgSlug<br>- agentId<br>- path | Scan duration in milliseconds |
```

Comment:
suggestion: would be more clear to include units into metric name, e.g. `AgentMonitoring/PerPathScanDurationMsec` and `AgentMonitoring/FileUploadLatencySec`